libs/Common/Octree.inl

```diff
@@ -167,6 +174,8 @@ void TOctree<ITEMARR_TYPE,TYPE,DIMS,DATA_TYPE>::_Insert(CELL_TYPE& cell, const P
 	// if this child cell needs to be divided further
 	if (bForceSplit || insertData.split(size, radius)) {
 		// init node and proceed recursively
+		cell.Release();
```

Comment:
cell should be always empty, pls remove the release and add an assert instead:
```
ASSERT(cell.m_child == NULL);
```

Reply:
![Selection_002](https://user-images.githubusercontent.com/2862064/198762557-7141df55-7a98-405e-8717-fc1c6afc9ef4.png)
![Selection_003](https://user-images.githubusercontent.com/2862064/198762569-67f4cc19-542c-48ae-8469-28d3434ecdea.png)

Reply:
I see, so pls fix the double allocation instead of just adding a release

Reply:
valgrind reported a memory leak. I see that the code does have new twice, but I'm not sure why.